src/invoice2data/extract/invoice_template.py

```diff
@@ -90,7 +93,10 @@ def prepare_input(self, extracted_str: str) -> str:
 
         # Specific replace
         for replace in self.options["replace"]:
-            assert len(replace) == 2, "A replace should be a list of exactly 2 elements."
+            assert len(replace) == 2, (
+                "Error in Template %s \nA replace should be a list of exactly 2 elements."
```

Comment:
The other 7 asserts don't use `\n` in messages, I'd suggest dropping `\n` from this one for compatibility.